src/prefect/infrastructure/provisioners/cloud_run.py

```diff
@@ -58,26 +61,54 @@ async def _run_command(self, command, *args, **kwargs):
 
     async def _check_for_gcloud(self):
```

Comment:
maybe `_verify_gcloud_installation_and_account` or `check_gcloud_ready` would be more descriptive with the new `gcloud auth` check in it?